pkg/authenticator/config/config.go

```diff
@@ -25,102 +26,124 @@ type Config struct {
 }
 
 const (
+	// DefaultClientCertPath (this comment added to satisfy linter)
```

Comment:
You could have used one comment at the head of this block (before the `const (` line) to handle all of the exported consts, but this works.